fake-files/output-expected/spraggins/spraggins.ims.json

```diff
@@ -5,7 +5,18 @@
     "x"
   ],
   "zarrConfig": {
-    "store": "https://s3.amazonaws.com/vitessce-data/0.0.24/master_release/spraggins/",
+    "store": "https://vitessce-data.storage.googleapis.com/0.0.24/master_release/spraggins",
     "path": "spraggins.ims.zarr"
+  },
+  "domain": [
```

Comment:
Can we also change https://github.com/hubmapconsortium/vitessce-data/blob/master/python/img_hdf5_reader.py#L195 while we do this one?